client/landing/stepper/hooks/use-is-site-big-sky-eligible.ts

```diff
@@ -22,41 +21,28 @@ export function useIsBigSkyEligible() {
 	);
 
 	const featureFlagEnabled = config.isEnabled( 'calypso/big-sky' );
-	const [ , experimentAssignment ] = useExperiment(
+	const [ isLoading, experimentAssignment ] = useExperiment(
```

Comment:
`const featureFlagEnabled = config.isEnabled( 'calypso/big-sky' );`

Can be hoisted above the function just like `bypassBigSkyExperiment`.